src/app/components/MediaLoader/configs/aresMedia.ts

```diff
@@ -96,6 +102,8 @@ export default ({
     playerConfig: {
       ...basePlayerConfig,
       autoplay: pageType !== 'mediaArticle',
+      ...(isAmp && { insideIframe: true }), // TO DO? - Extend to cover syndication
```

Comment:
Based on previous settings this should be extended this to cover syndicated assets. I'm using flags since (as far as I understand) we're not using an iframe for the future canonical implementation.

Is the only way to tell if an asset is being syndicated via the routing used? (Those listed in the dropbox paper). If so, I can create a flag based on this. Otherwise I'm not sure how to do this.

I can do some digging into the `insideIframe` setting. E.g. whether it causes an issue if we use it in the config and then we don't render inside an iframe. (I've linked the cookbook link for `insideIframe` to the ticket)

Reply:
Also with a 'syndicatedAsset' flag or similar we could add the `embeddedOffsite` setting. Cookbook describes it as 'Set this to true when creating iframe for embedding outside of the BBC (eg. syndication).'. It is not used on the news-av-player on Morph.

Reply:
I think we should just pass down a prop into the `MediaLoader` used inside `AvEmbedsPageLayout`. Something like `embedded` or similar. We can assume the `av-embeds` page type will always be iframed, either for syndication or AMP, so we can set `insideIframe: true` based on that prop we pass down.

Reply:
Cool thanks. I hadn't clocked that future ws/routes weren't later going to go through the `AvEmbedsPageLayout`. (If they are then I can add a check for this)

Reply:
Have added embedded flag